lib/go-tc/parameters.go

```diff
@@ -277,12 +277,15 @@ func ParamExists(id int64, tx *sql.Tx) (bool, error) {
 
 // ParamsExist returns whether parameters exist for all the given ids, and any error.
 // TODO move to helper package.
-func ParamsExist(ids []int64, tx *sql.Tx) (bool, error) {
-	count := 0
-	if err := tx.QueryRow(`SELECT count(*) from parameter where id = ANY($1)`, pq.Array(ids)).Scan(&count); err != nil {
-		return false, errors.New("querying parameters existence from id: " + err.Error())
+func ParamsExist(ids []int64, tx *sql.Tx) (bool, []int64, error) {
```

Comment:
It looks like the first return value - the boolean - is only false if the error is non-nil or there are non-existing IDs being returned. Which means it doesn't serve much purpose because it can be trivially inferred from those other return values.